website/js/NavBarNew.js

```diff
@@ -119,7 +119,7 @@ var NavBarNew = React.createClass({
         );
 
         return (
-            <div className={classNames("navbar-container", {"beta": this.state.isBeta})}>
+            <div className={classNames("navbar-container", {"beta": this.state.isBeta && false})}>
```

Comment:
Just checking, but since this expression will always be false, was it the intent to disable the beta switch?

Reply:
ah good catch, this was a temporary change for testing. will fix